message/templates/messages.html

```diff
@@ -0,0 +1,48 @@
+{% extends "main/base_template.html" %}
+{% load static %}
+
+{% block stylesheets %}
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<link rel="stylesheet" type="text/css" href="{% static 'css/chat.css' %}">
+
+{% endblock %}
+
+{% block content %}
+<div class="chatcontainer">
+  <div class="messaging">
+        <div class="inbox_msg">
+          <div class="inbox_people">
+            <div class="headind_srch">
+              <div class="recent_heading">
+                <h4>Recent Chats</h4>
+              </div>
+            </div>
+            <div class="inbox_chat">
+         {% for contact in contacts %}
+                <a href="/chat/{{contact.id}}"><div class="chat_list">
+                  <div class="chat_people">
+                      {% if user == 'D' %}
+                        <div class="chat_img"> <img src="{{contact.dog_picture_url}}" alt="nopic"> </div>
+                          <div class="chat_ib">
+                             <h5>{{ contact }}</h5>
+                          </div>
+                          {% else %}
+                           <div class="chat_img"> <img src="{{contact.get_daycare_primary_image_url}}" alt="nopic"> </div>
+                             <div class="chat_ib">
+                                <h5>{{ contact.name }}</h5>
+                              </div>
+                      {% endif %}
+
+                  </div>
+                </div>
+                </a>
+            {% endfor %}
+
+                    </tbody>
+                </table>
```

Comment:
Is there a table here? where is the opening tag?

Reply:
you're right, this is leftovers from a different try. 
Removed!